packages/utilities/src/customizations/Customizations.ts

```diff
@@ -32,18 +32,24 @@ let _allSettings = GlobalSettings.getValue<ICustomizations>(CustomizationsGlobal
 let _events: (() => void)[] = [];
 
 export class Customizations {
+  private static _suppressUpdates: boolean;
+
   public static reset(): void {
     _allSettings.settings = {};
     _allSettings.scopedSettings = {};
   }
 
-  // tslint:disable-next-line:no-any
+  /** Apply global Customization settings.
+   * @example Customizations.applySettings(\{ theme: \{...\} \});
```

Comment:
TSLint made me escape all these things... not sure if there's a better way to do it, the intellisense comments in VSCode don't look right